lib/http/2/stream.rb

```diff
@@ -86,7 +87,10 @@ def initialize(connection:, id:, weight: 16, dependency: 0, exclusive: false, pa
       @send_buffer = []
 
       on(:window) { |v| @remote_window = v }
-      on(:local_window) { |v| @local_window = v }
+      on(:local_window) do |v|
+        @local_window_size = v
+        @local_window = v
```

Comment:
@local_window_max? Both are tracking size, the names as is don't make it easy to grok what's going on.

Also, you can simplify this (and above) to: `@local_window_max = @local_window = v`

Reply:
is this `@local_window_max` supposed to be defined? What should I use it for? I thought that was the point of `@local_window_size`, hence the update there.